assets/map/art_map/customertest.tmx

```diff
@@ -25,16 +25,16 @@
 3,9,4,1,9,9,9,9,9,9,9,9,9,9,9,2147483649,0,0,0,0,
 0,0,0,27,0,0,0,0,0,0,0,0,0,0,0,2684354569,0,0,0,0,
 0,0,0,27,0,0,0,0,0,0,0,0,0,0,0,2684354569,0,0,0,0,
-0,0,0,27,0,0,3,21,3,8,14,28,0,0,0,2684354569,0,0,0,0,
-0,0,0,27,0,0,0,0,0,0,0,0,0,0,0,2684354569,0,0,0,0,
+0,0,0,27,0,0,3,21,3,8,14,28,36,0,0,2684354569,0,0,0,0,
+0,0,0,27,0,0,12,0,0,0,0,0,0,0,0,2684354569,0,0,0,0,
 0,0,0,27,0,0,0,0,0,0,0,0,0,0,0,2684354569,0,0,0,0,
 0,0,0,1610612745,0,0,22,0,0,32,0,32,0,0,0,2684354569,0,0,0,0,
-0,0,0,1610612745,0,0,0,0,0,0,0,0,0,0,0,2684354569,0,0,0,0,
+0,0,0,1610612745,0,0,12,0,0,0,0,0,0,0,0,2684354569,0,0,0,0,
 0,0,0,1610612745,0,0,0,0,0,0,0,0,0,0,0,2684354569,0,0,0,0,
 0,0,0,1610612745,0,0,34,34,0,26,25,4,0,0,0,2684354569,0,0,0,0,
 0,0,0,1610612745,0,0,0,0,0,0,0,0,0,0,0,2684354569,0,0,0,0,
 0,0,0,1610612745,0,0,0,0,0,0,0,0,0,0,0,2684354569,0,0,0,0,
-0,0,0,1610612745,0,0,0,0,0,0,0,0,0,0,0,2684354569,0,0,0,0,
+0,0,0,1610612745,0,0,35,0,0,0,0,0,0,0,0,2684354569,0,0,0,0,
```

Comment:
Tilemap edit has introduced a clipping issue (this may not be specifically on the line attached to the review comment).

`master` @ cffb2bdfb575912110c8d452fbf51d7667e745b4
![image](https://user-images.githubusercontent.com/49307226/224217270-69520a79-430f-47d4-ab70-e703e8b21e23.png)

`CreateDough` @ cb9a3e1886c7f4d0f0c3e50ecfc87aa47580c895
![image](https://user-images.githubusercontent.com/49307226/224217544-1adaf552-92ad-483b-8560-e822431b46df.png)